cmd/lncli/commands.go

```diff
@@ -2105,3 +2108,35 @@ func updateFees(ctx *cli.Context) error {
 	printRespJSON(resp)
 	return nil
 }
+
+var setAliasCommand = cli.Command{
+	Name:      "setalias",
+	Usage:     "set the alias of this node",
+	ArgsUsage: "alias",
+	Action:    actionDecorator(setAlias),
+}
+
+func setAlias(ctx *cli.Context) error {
+	ctxb := context.Background()
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+
+	var alias string
+
+	switch {
+	case ctx.IsSet("alias"):
```

Comment:
This looks good, but you must also add the `StringFlag` to the command definition :)

Reply:
Whoops, fixed!